### PR TITLE
chore: batch replace `http:` with `https:`

### PR DESCRIPTION
**Pull request by PassionPenguin**
see merged PR mdn/content#33376

related hash 94ef07a7b073c2663cbace0667bdb717a40bfa28

Thx :)

### CODE REVIEW

**Review by leon-win — APPROVED** [review]

**Comment by leon-win** [comment]
Good for `ru`, thank you @PassionPenguin ! :+1:

**Review by SphinxKnight — APPROVED** [review]
Thanks @PassionPenguin 🙇 Just nitpicking with a (non blocking) suggestion. / OK for `fr`

File: files/fr/learn/server-side/first_steps/website_security/index.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ XSS est un terme utilisé pour décrire une classe d'attaque qui permet à l'att
...
+  On prendra par exemple une fonction de recherche dans un site où les termes recherchés sont encodés en tant que paramètres dans l'URL, et que ces termes sont affichés en permanence avec les résultats. Un attaquant peut construire un lien de recherche contenant un script malicieux en tant que paramètre (ex: `https://developer.mozilla.org?q=beer<script%20src="http://example.com/tricky.js"></script>`) et le transmettre par e-mail à un autre utilisateur. Si l'utilisateur ciblé clique sur ce "lien intéressant", le script sera exécuté quand les résultats de la recherche seront affichés. Comme vu auparavant, cela donne à l'attaquant toute les informations dont il a besoin pour se connecter sur le site avec le compte de la victime — potentiellement faire des achats en tant que cet utilisateur ou accèder à la liste de contacts..
```
[the remark this answers is not in the document]
Comment (PassionPenguin) [reply files/fr/learn/server-side/first_steps/website_security/index.md:52]: that's good localization haha :) u're right!

**Review by JasonLamv-t — APPROVED** [review]
LGTM for `zh`, thank you! @PassionPenguin
/cc @yin1999

**Review by JuanVqz — APPROVED** [review]
Looks good for the `ES` team, thanks! cc @Graywolf9

**Review by mfuji09 — APPROVED** [review]
LGTM for `ja` contents.

**Review by yechoi42 — APPROVED** [review]

**Review by josielrocha — APPROVED** [review]
Approve for `pt-BR` 👍🏼

**Pull request merged**